src/main/scala/edu/washington/cs/seguard/js/JSFlowGraph.scala

```diff
@@ -235,7 +247,20 @@ object JSFlowGraph {
           abstractInstruction(node.getNode.getDU, symTable, instruction) match {
             case Some(u) => {
               // DefUse based analysis
-              dot.drawNode(u, NodeType.STMT)
+              var u_complete = u
```

Comment:
What does `u_complete` mean?

Reply:
It means the complete name of u. For example, when u is [get]env, and n_complete would be process[env]